pkg/apis/devconsole/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -25,94 +25,17 @@ import (
 )
 
 // DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
-func (in *Component) DeepCopyInto(out *Component) {
```

Comment:
You are removing deep copy for Component API? Make sure you have the latest master in your local repo when regenerating deep copy. It looks like it's missing Component API.

Reply:
because I generated the code within the `git-service` repository and I didn't copy the `component_types.go` there. 

The user experience is terrible - copy from types from `api` repo to `operator` repo to generate code that is needed in `git-service` repo. So then copy the generated code to `git-service` and every time when any type is changed copy it back to `operator` repo, regenerate and copy back... and then, finally, copy the resulting code to `api` repo...

Reply:
Yeah... It's not very dev friendly flow :) Maybe we should add some make targets to help with some tasks (like copying stuff around and cleaning it up with package replacement). Or another option would be just to merge everything to one mono repo ;)
But for now we just need to make sure we run `make generate` in devconsole-api repo after all these dance with moving apis.